inst/pages/beta_diversity.qmd

```diff
@@ -613,20 +613,8 @@ rda <- attr(rda, "rda")
 coef <- rda$CCA$v
 coef <- coef[, 1, drop = FALSE]
 
-# Get the taxa with biggest weights
-top_coef <- head(coef[rev(order(abs(coef))), , drop = FALSE], 20)
-# Sort weights in increasing order
-top_coef <- top_coef[order(top_coef), ]
-
-# Create data.frame
-df <- data.frame(
-    x = top_coef,
-    y = factor(names(top_coef), unique(names(top_coef))))
-
-# Create a plot
-ggplot(df, aes(x = x, y = y)) +
-    geom_bar(stat = "identity") +
-    labs(x = "", y= "", title = "Top Taxa") +
+#Create a plot
+plotLoadings(coef, ncomponents = 1, n = 20) + labs(x = "", y= "", title = "Top Taxa") +
```

Comment:
Hmm - why this doesn't use TreeSE directly (but coef)?

Reply:
It's because loadings are not stored in the "rotation" or "loadings" attribute (discussed in [#604](https://github.com/microbiome/mia/issues/604#issuecomment-2271153814) )
It requires a wrapper I believe?

Reply:
There is a small problem in addRDA as it should store the results to this slot https://github.com/microbiome/mia/issues/634

Let's keep this PR open until the mia is fixed